pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -48,13 +48,13 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) < 1 {
-				return errors.New("name of image is required")
+				return errors.New("a single image name or a space-separated list of image names is required")
```

Comment:
this is still inaccurate, the spaces are a detail of the shell.

Reply:
Ohh, so I suppose `a list of image names is required` message would be good. `a single image name` part feels redundant in my opinion but I'd love to hear your opinion.

Reply:
that SGTM, I think a list is fine, given that one image is just a list of only one image 👍